classes/tank.class.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class Tank{
+    public $life;
+    public $attack;
+    public $defence;
+    public $moves;
+    // TODO: id tank (variabila static in clasa abstracta?)
+
+    protected bool $isDefending = true;
+    protected $armorBonus = 20;
+
+    function __construct($life = 78, $attack = 65, $defence = 175, $moves = 1)
+    {
+        $this->life = $life;
+        $this->attack = $attack;
+        $this->defence = $defence;
+        $this->moves = $moves;
+    }
+
+    function __destruct()
+    {
+        echo '<p class="alert"> A tank was destroyed! </p>';
+    }
+\
```

Comment:
Can you please explain me the purpose of this \ ?

Reply:
That is for sure a typo